cvat-canvas/src/typescript/canvasModel.ts

```diff
@@ -651,6 +651,15 @@ export class CanvasModelImpl extends MasterImpl implements CanvasModel {
     }
 
     public highlight(clientIDs: number[], severity: HighlightSeverity | null): void {
+        if (clientIDs.length !== 0) {
+            const states = clientIDs.map((id) => (
+                this.objects.find((_state: any): boolean => _state.clientID === id)
+            ));
+            if (states.some((state) => !state)) {
+                return;
+            }
```

Comment:
Does this code mean that if we want to highlight elements 1,2,3 and there is not element 3, we will not highlight anything?